pennylane/__init__.py

```diff
@@ -102,7 +103,7 @@
 
 # collections needs to be imported after all other pennylane imports
 from .collections import QNodeCollection, dot, map, sum
-import pennylane.grouping  # pylint:disable=wrong-import-order
+import pennylane.pauli.grouping as grouping  # pylint:disable=wrong-import-order
```

Comment:
We use the grouping functions in the pauli grouping module instead. The original grouping module is still kept, but not directly imported.